CONTRIBUTING.md

```diff
@@ -131,11 +145,13 @@ each pull request contains just one commit.
 
   6. If everything was successful you can push your changed history
      back up to GitHub:
-
-        $ git push --force-with-lease
+     ```shell
+     $ git push --force-with-lease
+     ```
 
 [xmonad]: https://github.com/xmonad/xmonad
 [xmonad-contrib]: https://github.com/xmonad/xmonad-contrib
 [xmonad-testing]: https://github.com/xmonad/xmonad-testing
 [x11]: https://github.com/xmonad/X11
 [ml]: https://mail.haskell.org/cgi-bin/mailman/listinfo/xmonad
+[xmonad-doc-developing]: http://hackage.haskell.org/package/xmonad-contrib-0.16/docs/XMonad-Doc-Developing.html
```

Comment:
Make this `http://hackage.haskell.org/package/xmonad-contrib/docs/XMonad-Doc-Developing.html`, hackage will autoredirect to the latest version.

Reply:
Thanks for the tip!